src/ResizeSensor.js

```diff
@@ -171,16 +183,16 @@
                 reset();
             };
 
-            var addEvent = function(el, name, cb) {
+            var addPassiveEvent = function(el, name, cb) {
                 if (el.attachEvent) {
                     el.attachEvent('on' + name, cb);
                 } else {
-                    el.addEventListener(name, cb);
+                    el.addEventListener(name, cb, passiveEvents);
```

Comment:
Is there an error when we always pass `{passive: true}` in it, even if the browser doesn't support it?